sn_cli/src/subcommands/files/chunk_manager.rs

```diff
@@ -430,6 +441,7 @@ impl ChunkManager {
     fn read_file_chunks_dir(
         file_chunks_dir: PathBuf,
         path_xor: &PathXorName,
+        original_file_path: PathBuf,
         original_file_name: OsString,
```

Comment:
the `original_file_path` contains the `file_name` ?

Reply:
yes, the file_name is kept for convenience since it needs to be resolved from an `Option<OsStr>`